### PR TITLE
Develop core functionality of calculator

### PR DESCRIPTION
**Pull request by Becky449**
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, I created a React component that matches the instructed design.
- Don't add any extra design.
- Developed just the UI of the calculator; the math logic to make it work will be added in the next project.
- Used class-based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it renders without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
```
Comment (elmar8287) [inline src/App.js:3]: - [ ] Please, consider removing all the boilerplate that got created by the CRA (texts, images, and styles) - just clear your folder structure 👍 

image

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
```
Comment (elmar8287) [inline src/App.js:2]: - [ ] Please, consider that you are not using this App.css style file and all styles inside accordingly. So, as it came from the CRA command, consider removing this style (in your case whole App.css as a file) 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Becky449.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good PR description 👍 
- No linter issues

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] Please, consider fixing the broken link to the MIT.md file

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Becky449 , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive PR title & description.
:heavy_check_mark: Well-implemented suggested changes.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**